coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/functions/HiveFunctionResolver.java

```diff
@@ -218,7 +222,7 @@ public void addDynamicFunctionToTheRegistry(String funcClassName, Function funct
             new SqlUserDefinedFunction(new SqlIdentifier(funcClassName, ZERO),
                 new HiveGenericUDFReturnTypeInference(funcClassName, hiveTable.getDaliUdfDependencies()), null,
                 createSqlOperandTypeChecker(numOfOperands), null, null),
-            hiveTable.getDaliUdfDependencies(), functionName));
+            hiveTable.getDaliUdfDependencies(), functionName, funcClassName));
```

Comment:
At some point we deal with `functionName` as a class name (e.g., `String unversionedClassName = getName();`). Is there a way to stick to one terminology througout the whole code?

Reply:
Modified to use `functionName` to refer to the function name in view text, and `funcClassName` to refer to the function class.